packages/amplify-graphiql-explorer/src/ClearDataModal.tsx

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import { Button, Modal } from "semantic-ui-react";
+
+type Props = {
+  onClose: Function;
+  onClear: Function;
+};
+export function ClearDataModal(props: Props) {
+  const [isOpen, setIsOpen] = useState(true);
+  const actionText = "Clear";
+  const cancelText = "Cancel";
+  const confirmationText = "Do you want to clear the data of your mock server?";
```

Comment:
These can be pulled out of the definition, since they don't need to be redefined on every instance of this component.